manifests/k8s-cluster-role-binding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: external-mdns-viewer
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: external-mdns
+subjects:
+- kind: ServiceAccount
+  name: external-mdns
+  namespace: default # Assicurati che il namespace sia corretto
```

Comment:
I created PR #31 which I _think_ will help address this. The namespace can be configured in `manifests/rbac/kustomize.yaml` and will propagate to all of the relevant resources when applying the manifests with `kubectl apply --kustomize manifests/rbac/`.